app/assets/stylesheets/modules/posts.scss

```diff
@@ -4,10 +4,23 @@
   text-align: left;
   width: 70%;
 
-  .posts-search-title {
-    margin: 25px 50px 0;
+  .posts-search {
+    border-top: 1px solid $gray-medium-light;
+    color: $gray-medium-light;
+    margin-top: 35px;
+    padding: 50px;
+    text-align: center;
 
-    @include mq-mobile() { margin: 25px 20px 0; }
+    a { color: $gray-medium-light; }
```

Comment:
I would change this selector for a class selector maybe `.posts-search-link` or something like that